packages/snowpack/src/config.ts

```diff
@@ -228,10 +232,6 @@ const configSchema = {
         treeshake: {type: 'boolean'},
         installTypes: {type: 'boolean'},
         sourceMap: {oneOf: [{type: 'boolean'}, {type: 'string'}]},
-        alias: {
-          type: 'object',
-          additionalProperties: {type: 'string'},
-        },
```

Comment:
I’m -1 on removing this here; can we just add a TODO and a warning that people should use `alias` instead? We should probably check below just so we don’t break config.

Reply:
sgtm!